pkg/oonimkall/taskrunner_test.go

```diff
@@ -270,6 +277,15 @@ func TestTaskRunnerRun(t *testing.T) {
 					return "GARR"
 				},
 			},
+
+			Loader: &mocks.ExperimentTargetLoader{
+				MockLoad: func(ctx context.Context) ([]model.ExperimentTarget, error) {
+					targets := []model.ExperimentTarget{
+						model.NewOOAPIURLInfoWithDefaultCategoryAndCountry(""),
+					}
+					return targets, nil
+				},
+			},
```

Comment:
By default we mock an experiment taking an empty entry (like dash, ndt, telegram, etc.)